utils/python/CIME/XML/env_batch.py

```diff
@@ -4,7 +4,7 @@
 from standard_module_setup import *
 
 from env_base import EnvBase
-
+from CIME.utils import convert_to_string
```

Comment:
These were just to get tests working and when we merge to master we want to make sure that the master copy of env_batch and not this one is kept.